docs/tutorial/process-model.md

```diff
@@ -214,8 +214,22 @@ This feature is incredibly useful for two main purposes:
   URL, you can add custom properties onto the renderer's `window` global that can
   be used for desktop-only logic on the web client's side.
 
+## The utility process
+
+Each Electron app can spawn multiple child processes from the main process using
+the [UtilityProcess][] API. The utility process runs in a Node.js environment,
+meaning it has the ability to `require` modules and use all of Node.js APIs.
+The utility process can be used to host for example: untrusted services,
+CPU intensive tasks or crash prone components which would have previously
+been hosted in the main process or process spawned with Node.js child_process module.
+The primary difference between the utility process and process spawned by Node.js
+child_process module is that the utility process can establish a communication
+channel with a renderer process using [`MessagePort`][]s.
```

Comment:
Is there any advantage to using `child_process` over `UtilityProcess`, or can `UtilityProcess` be considered strictly superior due to its MessagePort support?

Reply:
Currently `UtilityProcess` only offers the `fork` variant of `child_process` and for those cases you can always prefer the `UtilityProcess`. If you want to do `exec` or `spawn` then it is still preferred to use `child_process` module.